docker/couchdb/Dockerfile

```diff
@@ -1,8 +1,20 @@
-FROM couchdb:3.2.2
+FROM cmssw/filebeat:latest as filebeat
+FROM couchdb:3.4.1
 MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
 
-RUN apt-get update && apt-get install -y vim less sudo wget unzip python pip
-RUN pip install CMSCouchapp
+RUN apt-get update && apt-get install -y vim less sudo wget unzip python3 pip
+# NOTE:
+# RUN pip install CMSCouchapp
+# when we install CMSCouchapp on Debian system it fails with PEP 668 error
+# which prevents installing external (Python packages from pip) on Debian system
+# there are two ways to resolve it:
+# - option A: install with (potential) break of system packages
+#   RUN pip install --break-system-packages CMSCouchapp
+# - option B: install in virtual environment
+RUN apt install -y python3.11-venv
```

Comment:
Why python3.11? Is it the base python version that gets installed with `apt-get`?

BTW, do we still have to install python? Or does it come with the CouchDB vanilla image?